log/log.go

```diff
@@ -266,3 +270,18 @@ func extractCallerDetails() (file string, line int, pkg string, function string,
 
 	return "", 0, "", "", errors.New("unable to extract the caller details")
 }
+
+// getDefaultLogLevel extracts the log level out of the ENV variable. It is used
+// in tests and as default static initialization of the log. If the ENV variable
+// is not set then the log level is Info.
+func getDefaultLogLevel() log.Level {
```

Comment:
@alexeykazakov we have it already but if the env variable isn't defined then we need to set a level :)